src/Bicep.Core/TypeSystem/ResourceTypeRegistrar.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using Bicep.Core.Resources;
+using Bicep.Core.TypeSystem.Azrm;
+
+namespace Bicep.Core.TypeSystem
+{
+    public class ResourceTypeRegistrar
```

Comment:
>ResourceTypeRegistrar [](start = 17, length = 21)

I think this will also help the OAM folks to do their experiments. I think @rynowak was asking at some point if it's possible to change what properties are available at the top level.